docs/tutorials/stateful-application/cassandra.md

```diff
@@ -1,374 +1,134 @@
-| CASSANDRA_RPC_ADDRESS | 0.0.0.0          |
-
-## Quickstart
-
-If you want to jump straight to the commands we will run,
-here are the steps:
-
-```sh
-#
-# StatefulSet
-#
+{% capture overview %}
+This tutorial will show you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this instance, a custom Cassandra `SeedProvider` enables Cassandra to discover new Cassandra nodes as they join the cluster.
+{% endcapture %}
 
-# clone the example repository
-git clone https://github.com/kubernetes/examples
-cd examples
+{% capture objectives %}
+* Create and Validate a Cassandra headless service.
+* Use a StatefulSet to create a Cassandra ring.
+* Validate the StatefulSet.
+* Modify the StatefulSet.
+* Delete the StatefulSet and its pods.
+{% endcapture %}
 
-# create a service to track all cassandra statefulset nodes
-kubectl create -f cassandra/cassandra-service.yaml
+{% capture prerequisites %}
+You should already have a basic familiarity with [Pods](#), [Services](#), and [StatefulSets](#), but to complete this tutorial you only need: 
 
-# create a statefulset
-kubectl create -f cassandra/cassandra-statefulset.yaml
+* To [Install and Configure](/docs/tasks/tools/install-kubectl/) the `kubectl` command line
 
-# validate the Cassandra cluster. Substitute the name of one of your pods.
-kubectl exec -ti cassandra-0 -- nodetool status
+* To have a v1.5 or later Kubernetes Cluster running
 
-# cleanup
-grace=$(kubectl get po cassandra-0 -o=jsonpath='{.spec.terminationGracePeriodSeconds}') \
-  && kubectl delete statefulset,po -l app=cassandra \
-  && echo "Sleeping $grace" \
-  && sleep $grace \
-  && kubectl delete pvc -l app=cassandra
+Check out the the [getting started guides](/docs/setup/pick-right-solution/) if you don’t already have a cluster. 
 
-#
-# Resource Controller Example
-#
-
-# create a replication controller to replicate cassandra nodes
-kubectl create -f cassandra/cassandra-controller.yaml
-
-# validate the Cassandra cluster. Substitute the name of one of your pods.
-kubectl exec -ti cassandra-xxxxx -- nodetool status
+Warning: [Minikube](/docs/getting-started-guides/minikube/) defaults to 1024MB of memory and 1 CPU which results in an insufficient resource errors. 
 
-# scale up the Cassandra cluster
-kubectl scale rc cassandra --replicas=4
+To avoid these errors add the flags `--memory 5120` and `--cpus=4` to `minikube start`
 
-# delete the replication controller
-kubectl delete rc cassandra
-
-#
-# Create a DaemonSet to place a cassandra node on each kubernetes node
-#
-
-kubectl create -f cassandra/cassandra-daemonset.yaml --validate=false
-
-# resource cleanup
-kubectl delete service -l app=cassandra
-kubectl delete daemonset cassandra
 ```
+minikube start --memory 5120 --cpus=4
+``` 
+* To download [cassandra-service.yaml](docs/tutorials/stateful-application/cassandra-service.yaml) and [cassandra-statefulset.yaml](/docs/tutorials/stateful-application/cassandra-statefulset.yaml)
```

Comment:
Dead links and a confusing sentence.  Not sure what this is asking me to do. 
Also, in general, avoid single bullet points.